backend/app/management/commands/syncdb.py

```diff
@@ -92,21 +95,27 @@ def handle(self, *args, **options):
         # THIS IS JUST FOR PROTOTYPING NEVER EVER EVER EVER IN PRODUCTION do this
         superuser = User.objects.create_superuser('admin', password='adminadmin')
 
-        if not values:
+        if not databases:
```

Comment:
In cases like this where there's an exit condition and then you run all of the business logic otherwise, my preference is this style (once it gets long enough... which it has at this point!):

```
if not databases:
    print_header('No data found')
    return

# the rest of your code goes here...
```